### PR TITLE
build: support node18

### PR DESCRIPTION
**Pull request by vincent-maverick**
sad: `$ yarn add -D cz-format-extension --ignore-engines`
happy: `$ yarn add -D cz-format-extension`

I believe node v18 is working fine, but I don't know what the problem is.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,6 @@
     "typescript": "^4.5.5"
   },
   "engines": {
-    "node": "^12 || ^14 || ^16 || ^17"
+    "node": "^12 || ^14 || ^16 || ^17" || ^18
```
Comment (vincent-maverick) [inline package.json:69]: oh my mistake...
Must be written in double quotes...
I hope the intention is conveyed.

Comment (vincent-maverick) [reply package.json]: https://github.com/tyankatsu0105/cz-format-extension/pull/109/commits/6da47309501b1f9b0ff5e34b54e2491116aefa6c

done!

**Comment by tyankatsu0105** [comment]
@vincent-maverick 
Thanks for the PR!
I'd like to add `18` to https://github.com/tyankatsu0105/cz-format-extension/blob/main/.github/workflows/test.yml#L12 and https://github.com/tyankatsu0105/cz-format-extension/blob/main/.github/workflows/type-check.yml#L12 :)

**Comment by vincent-maverick** [comment]
[Update test.yml](https://github.com/tyankatsu0105/cz-format-extension/pull/110)
[Update type-check.yml](https://github.com/tyankatsu0105/cz-format-extension/pull/111)

done!

**Comment by vincent-maverick** [comment]
@tyankatsu0105 
Or according to commitizen
```
"engines": {
  "node": ">= 12"
},
```
It may be good as well.
Just an idea.

**Review by tyankatsu0105 — APPROVED** [review]

**Pull request merged**